Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -2898,7 +2898,7 @@ public void moveSelectedPagesDown() {
             newSelectionList.add(String.valueOf(pageIndex + 1));
         }
 
-        allPagesSelection = newSelectionList.toArray(new String[newSelectionList.size()]);
+        allPagesSelection = newSelectionList.toArray(new String[0]);
```

Comment:
Why did you changes this? Amount of entries is well known.

Reply:
Creating array without size is more efficient since java 6.